src/main/java/edu/indiana/dlib/amppd/service/impl/FileStorageServiceImpl.java

```diff
@@ -209,4 +221,42 @@ else if (supplement instanceof PrimaryfileSupplement) {
 		return dirname + File.separator  + filename;
 	}
 
+	/**
+	 * @see edu.indiana.dlib.amppd.service.FileStorageService.uploadFileToGalaxy(String,String)
+	 */
+	public ClientResponse uploadFileToGalaxy(String filePath, String libraryName) {
```

Comment:
Would it make more sense to return boolean or custom response, instead of `ClientResponse` so its easier to implement?